src/EntityFramework.SqlCe.Legacy.nuspec

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>EntityFramework.SqlCe.Legacy</id>
```

Comment:
We were wonder what you'd call the package. One thought we had was something like `EntityFramework7.SqlServerCe35` and maybe pair it with `EntityFramework7.SqlServerCe40`.